packages-content-model/roosterjs-content-model-dom/lib/modelApi/creators/createTableCell.ts

```diff
@@ -27,6 +29,6 @@ export function createTableCell(
         spanLeft,
         spanAbove,
         isHeader: !!isHeader,
-        dataset: {},
+        dataset: { ...dataset } ?? {},
```

Comment:
Actually, just do "{...dataset}" is ok, no need to add " ?? {}" since it must not be null.